app/src/main/java/net/gsantner/opoc/frontend/GsSearchOrCustomTextDialog.java

```diff
@@ -94,18 +94,16 @@ public static class DialogOptions {
         public boolean isSoftInputVisible = true;
         public boolean isDismissOnItemSelected = true;
         public int listPosition = -1;
-        public int gravity = Gravity.NO_GRAVITY;
         public int dialogWidthDp = WindowManager.LayoutParams.MATCH_PARENT;
         public int dialogHeightDp = WindowManager.LayoutParams.WRAP_CONTENT;
         public int searchInputType = InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS;
-        public float[] portraitAspectRatio = {0.0f, 1.0f};
-        public float[] landscapeAspectRatio = {0.0f, 1.0f};
```

Comment:
@guanglinn do we really need this? I will leave it in if you think this is important.